tests/PySys/local_mqtt/local_mqtt_pub_sub_retain/run.py

```diff
@@ -0,0 +1,78 @@
+from pysys.basetest import BaseTest
+
+import time
+
+"""
+Validate local publishing with retain flag and subscribing:
+
+Given a configured system
+When we publish a message with retain flag to temperature sensor alaram topic (simulate set alaram)
+When we publish an empty message with retain flag to pressure sensor alarm topic (simulate clear alarm)
+When we start tedge sub in the background to subscribe to tempearture sensor alarm topic
+When we start tedge sub in the background to subscribe to pressure sensor alarm topic
+Then we kill both the subscribers
+Then we find the retained message in the output of temperature subscriber log
+And no message in the pressure sensor subscriber log
```

Comment:
A green result has little value here, since it will be green even with a command that doesn't even connect to MQTT.